server/public/styles.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: 'roboto', sans-serif;
+}
+
+h1 {
+  font-family: 'poppins', sans-serif;
+  font-weight: 400;
+  font-size: 36px;
+  line-height: 36px;
+  letter-spacing: -2px;
+  margin: 0;
+  padding-bottom: 10px;
+}
+
+.container {
+  display: flex;
+}
+
+.menu-container {
+  position: fixed;
+  display: flex;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.8);
+}
+
+.menu-content {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  width: 250px;
+  height: 100%;
+  top: 0;
+  padding-left: 2rem;
+  border-radius: 0 5% 5% 0;
+}
+
+a {
+  text-decoration: none;
+  padding-top: 1rem;
+}
+
+.fa-bars {
+  font-size: 24px;
+  padding: 0.5rem;
+}
```

Comment:
It's not good to style CSS classes you didn't define. Instead add a custom class.